src/main/java/MiniMax.java

```diff
@@ -1,5 +1,94 @@
 public class MiniMax {
-    public int treeScore(Node node) {
-        return node.getScore();
+    private final Board board;
+    private Rules rules = new Rules();
+    private String maximizer;
+    private String minimizer;
+
+    public MiniMax(Board board) {
+        this.board = board;
+    }
+
+    private Integer evalBoard(int depth) {
+        final int WINNING_SCORE = 10;
+        final int LOSING_SCORE = -10;
+        final int DRAW_SCORE = 0;
+        if (rules.victoryCheck(board.currentMoves(), maximizer).equals(maximizer)){
```

Comment:
can we refactor these if statements into just one if statement?

Reply:
moved into one if/else if/else statement.